bindings/python/pinocchio/display/abstract_display.py

```diff
@@ -0,0 +1,70 @@
+from .. import libpinocchio_pywrap as pin
+from ..shortcuts import buildModelsFromUrdf, createDatas
+
+import time
+
+class AbstractDisplay(object):
+    """Pinocchio displays are employed to easily display a model at a given configuration.
+    AbstractDisplay is not meant to be directly employed, but only to provide a uniform interface and a few common methods.
+    New displays should extend this class and override its methods as neeeded.
+    """
+
+    @classmethod
+    def BuildFromURDF(cls,filename, package_dirs=None, root_joint=None, verbose=False, meshLoader=None):
+        """Construct a display from the given URDF file"""
+
+        display = cls()
+        display.initFromURDF(filename, package_dirs, root_joint, verbose, meshLoader)
+        return display
+
+    def initFromURDF(self,filename, package_dirs=None, root_joint=None, verbose=False, meshLoader=None):
+        """Initialize a display from the given URDF file"""
+
+        model, collision_model, visual_model = buildModelsFromUrdf(filename, package_dirs, root_joint, verbose, meshLoader)
+        cls = type(self)
+        cls.__init__(self,model=model,collision_model=collision_model,visual_model=visual_model)
+
+    def __init__(self, model = pin.Model(), collision_model = None, visual_model = None):
+        """Construct a display from the given model, collision model, and visual model"""
+
+        self.model = model
```

Comment:
Should we copy the model here? I think so.

Reply:
I don't see why. This is not what we were doing in `RobotWrapper` anyway. The model object is basically immutable, for the use we make of it. This way we can share some information. I thought this was one of the reasons for the whole idea of splitting model and data in the first place. Notice the data objects are not shared, but they are created with the viewer.

Reply:
Yes, but RobotWrapper is not well done. As you are working on the splitting, we should do it completely. 
The splitting between model and data goes beyond this scope, it is very helpful for performances issues.
Here, I would like to think twice before merging how far the memory used for the viewer has to be shared with the rest of the application. One thing to add as the last argument should be a copy option with a default value to False I think to really allow a strict splitting between all the functionalities.

Reply:
In Python, it is absolutely normal to share objects through the same pointers.
Meh, I do not think the option would add a lot. If one *really* wants the display to use a copy of the models, they could simply pass copies in input, like this
```
display = GepettoDisplay(model.copy(),collision_model.copy(),visual_model.copy())
```

Actually, I see geometry models are *not* copyable in Python. I will make them copyable.

Reply:
I do not think adding this option is hard and may save some times and energy beforehand. 
I think this is a feature requested to really make both independent, without having to use the signature you've just suggested.

Reply:
I think this is somehow unusual for Python. However, if you want it, it will be easy to add. Still requires to expose `.copy()` to `GeometryModel` though

Reply:
This is for instance how NumPy works too.

Reply:
I added an argument called `copy_models`. I did the same for `setDisplay` in `RobotWrapper`